node/src/chain_spec.rs

```diff
@@ -180,6 +181,10 @@ fn development_genesis(
                 .collect(),
         },
         pallet_sudo: vanilla_runtime::SudoConfig { key: root_key },
+		orml_oracle_Instance2: VanillaOracleConfig {
```

Comment:
orml_oracle_Instance2 and orml_oracle_Instance1 are not reasonable names.

Reply:
Fixed: use orml_oracle_Instance1 only.